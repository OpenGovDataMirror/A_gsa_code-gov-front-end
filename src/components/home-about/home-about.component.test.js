import React from 'react'
import { shallow } from 'enzyme'

import HomeAbout from 'components/home-about/home-about.component'

const props = {
  mission: 'test-mission',
  vision: 'test-vision'
}

let wrapper
describe('components - HomeAbout', () => {
  beforeEach(() => {
    wrapper = shallow(<HomeAbout {...props} />)
  })

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
