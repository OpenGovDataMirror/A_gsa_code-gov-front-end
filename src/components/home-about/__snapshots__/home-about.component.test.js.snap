// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components - HomeAbout should render correctly 1`] = `
<section
  className="tablet:grid-col-6"
  id="about"
>
  <div
    className="bg-primary-lighter height-full"
  >
    <header
      className="padding-x-4 padding-top-6"
    >
      <h2
        className="font-heading-lg text-primary-darker text-center"
      >
        About Code.gov
      </h2>
      <p />
    </header>
    <ul
      className="padding-x-4"
    >
      <li
        className="padding-y-3"
      >
        <h2
          className="text-primary-darker text-uppercase tablet:margin-left-9 margin-bottom-0"
        >
          Vision
        </h2>
        <span
          className="display-flex tablet:flex-row margin-0"
        >
          <p
            className=""
          >
            test-vision
          </p>
          <img
            alt=""
            className="order-first height-7 margin-top-3 margin-right-2 display-none tablet:display-block"
            src="/assets/img/icons/vision_icon_sm.png"
          />
        </span>
      </li>
      <li>
        <h2
          className="text-primary-darker text-uppercase tablet:margin-left-9 margin-bottom-0"
        >
          Mission
        </h2>
        <span
          className="display-flex table:flex-row margin-0"
        >
          <p>
            test-mission
          </p>
          <img
            alt=""
            className="order-first height-7 margin-top-3 margin-right-2 display-none tablet:display-block"
            src="/assets/img/icons/mission_icon_sm.png"
          />
        </span>
      </li>
    </ul>
  </div>
</section>
`;
